bin/ream/src/cli/mod.rs

```diff
@@ -33,6 +38,26 @@ pub struct NodeConfig {
     )]
     pub network: Arc<NetworkSpec>,
 
+    /// HTTP port number
+    #[arg(long, default_value_t = DEFAULT_HTTP_PORT)]
+    pub http_port: usize,
+
+    /// HTTP bind address
+    #[arg(long,default_value_t = DEFAULT_HTTP_ADDRESS.to_string())]
+    pub http_address: String,
+
+    /// Allow CORS
+    #[arg(long, default_value_t=DEFAULT_HTTP_ALLOW_ORIGIN)]
+    pub http_allow_origin: bool,
+
+    /// discv5 listening port
+    #[arg(long, default_value_t = DEFAULT_DISCOVERY_PORT)]
+    pub discovery_port: u16,
+
+    /// disable discovery
```

Comment:
```suggestion
    #[arg(long, default_value_t = DEFAULT_HTTP_PORT)]
    pub http_port: usize,

    #[arg(long,default_value_t = DEFAULT_HTTP_ADDRESS.to_string())]
    pub http_address: String,

    #[arg(long, default_value_t=DEFAULT_HTTP_ALLOW_ORIGIN)]
    pub http_allow_origin: bool,

    #[arg(long, default_value_t = DEFAULT_DISCOVERY_PORT)]
    pub discovery_port: u16,

```

could you remove these doc comments and if you want to add a description please use the `help = "Choose mainnet, holesky, or sepolia",` help arg, as shown in a few of the clis above